src/index.html

```diff
@@ -6,9 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Moyo logo"
+        class="logo__image">
+      </a>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="link button">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
```

Comment:
You need to add naming classes with BEM
<img width="1081" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/18e077ac-e3ac-4d1a-b9e2-56cd52d8dc74">
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
```